packages/widget_toolkit_otp/README.md

```diff
@@ -0,0 +1,98 @@
+# Prime SMS Code
+
+`Prime SMS Code` package contains a number of useful widgets that can help you in building sms/pin 
```

Comment:
Please add a .gif

Reply:
Do not forget to add the assets in pubspec as well 

Have a look here for an example:
https://github.com/Prime-Holding/widget_toolkit/blob/d4b61a3dd2e53ece58b069f4b2e988ed3729f4f0/packages/widget_toolkit/pubspec.yaml#L71

Reply:
Also please add the new package to the sections
- Packages
- Documentation
- Samples